openwisp_radius/base/models.py

```diff
@@ -1085,6 +1088,9 @@ class AbstractOrganizationRadiusSettings(UUIDModel):
         blank=True,
         choices=OPTIONAL_FIELD_CHOICES,
     )
+    registration_api_enabled = models.BooleanField(
+        null=True, blank=True, default=True, help_text=_GET_MOBILE_PREFIX_HELP_TEXT,
```

Comment:
looks like `_GET_MOBILE_PREFIX_HELP_TEXT` should be changed to `_GET_REGISTRATION_ENDPOINT_HELP_TEXT`